rj_gameplay/stp/skill/ball_carrier.py

```diff
@@ -0,0 +1,37 @@
+from abc import ABC, abstractmethod
+from typing import Optional, Type
+
+import numpy as np
+
+import stp.skill as skill
+from stp import role as role
+
+
+class Pass:
+    __slots__ = ["passer", "receiver", "pt"]
+
+    passer: int
+    receiver: int
+    pt: np.ndarray
+
+    def __init__(self, passer: int, receiver: int, pt: np.ndarray):
+        self.passer = passer
+        self.receiver = receiver
+        self.pt = pt
+
+
+class IBallCarrier(skill.ISkill, ABC):
+    @abstractmethod
+    def get_pass(self) -> Optional[Pass]:
```

Comment:
I'm pretty sure this belongs at the tactic level. The functionality exposed should be `do_pass(self, pass: Pass)` and `do_shoot(self, shot: Shot)`.

Reply:
This isn't really meant to be an actual working play, but more as like something to think about how the library should look like in terms of ergonomics and stuff.

Reply:
How would `do_pass` and `do_shoot` work? Why does it belong at the tactic level?